chipsec/logger.py

```diff
@@ -395,6 +396,32 @@ def logger():
     """Returns a Logger instance."""
     return _logger
 
+def minimum_column_widths(table_data):
+    clean_data = clean_data_table(table_data)
+    column_widths = get_column_widths(clean_data)
+    min_widths = find_min_col_widths(column_widths)
+    return tuple(min_widths)
+
+def clean_data_table(data_table):
+    clean_table = [extract_column_values(row) for row in data_table]
+    return clean_table
+
+def extract_column_values(row_data):
+    clean_row = [row_data[0]]
```

Comment:
A cleaner way to do this would be: 
```suggestion
    clean_row = row_data[:1]
```